proposed/extended-coding-style-guide.md

```diff
@@ -107,14 +107,15 @@ There MUST NOT be more than one statement per line.
 
 ### 2.4 Indenting
 
-Code MUST use an indent of 4 spaces, and MUST NOT use tabs for indenting.
+Code MUST use an indent of 4 spaces for each indent level, and MUST NOT use
+tabs for indenting.
 
-### 2.5 Keywords and True/False/Null
+### 2.5 Keywords and Types
 
 PHP [keywords][] MUST be in lower case.
 
 The PHP types and keywords `int`, `true`, `object`, `float`, `false`, `mixed`,
-`bool`, `null`, `numeric`, `string` and `resource` MUST be in lower case
+`bool`, `null`, `numeric`, `string`, `void` and `resource` MUST be in lower case.
```

Comment:
Also, what about supporting `scalar` too? I don't see it about much, but php does have an `is_scalar` method.